meson.build

```diff
@@ -82,7 +78,7 @@ if gnutls.found()
 	config.set('ENABLE_TLS', true)
 endif
 
-if host_system == 'linux' and cc.has_header('sys/sdt.h')
```

Comment:
Rather than disable I want to add a proper option because `sys/sdt.h` is from a package `dev-util/systemtap` and the user should have an option when to disable a functionality. At gentoo everything is compiled so it is better to know what things are being built in. It is better to have an option to use a package rather than just using it if it is present and ignoring otherwise. 

I will add the check back but I would still like to have the `systemtap` option as well.

Reply:
Keep in mind that building with dtrace probes does not make systemtap or dtrace a dependency. The probes are instructions that are compiled into the code and do not affect the performance unless they are placed in silly places such as inside a loop that does a lot of processing.

Anyhow, if this is to become an optional inclusion, the it should follow the same pattern as other features which can be enabled such as tls and tight encoding. The way to do that is probably to use `declare_dependency` for systemtap and then do the same thing for it as for the other two optional dependencies.